app/fonts/stylesheets/home.scss

```diff
@@ -0,0 +1,21 @@
+@font-face {
+    font-family: 'fairviewregular';
+    src: url('fairview_regular-webfont.eot');
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings